api/products.js

```diff
@@ -66,6 +66,13 @@ class ProductsApi extends Api {
     return r && r.body
   }
 
+  async updateProduct (product, opts) {
+    opts = opts || {}
+    const request = this.pick(product, 'id', 'name', 'slug', 'version', 'productCategory')
```

Comment:
Same comments as `updateCustomer` (`opts` line not needed, add `externalOrg` and `externalKey `).

Also, it looks like the api expects the following properties:
- `version`
- `productCode` (instead of `slug`)
- `displayName` (instead of `name`)
- `description` (missing)
- `productCategory`

which aligns with the props listed in `createProduct`.